src/renderer/services/const.ts

```diff
@@ -2,7 +2,31 @@ import { PricePoolAssets, PoolAsset } from '../views/pools/types'
 import { Network } from './app/types'
 
 // Whitelist of pools for pricing things
-export const PRICE_POOLS_WHITELIST: PricePoolAssets = [PoolAsset.BTC, PoolAsset.ETH, PoolAsset.TUSDB]
+export const PRICE_POOLS_WHITELIST: PricePoolAssets = [PoolAsset.BTC, PoolAsset.ETH, PoolAsset.BUSDB]
 
 export const DEFAULT_NETWORK: Network = 'testnet'
 export const AVAILABLE_NETWORKS: Network[] = ['testnet', 'chaosnet', 'mainnet']
+
+const commonAssets = {
+  BNB: 'BNB.BNB',
+  ETH: 'ETH.ETH',
+  BTC: 'BTC.BTC',
+  BUSDB: 'BNB.BUSDB-000'
+}
+
+const networkPools = {
+  testnet: {
+    ...commonAssets,
+    RUNE: PoolAsset.RUNE67C
+  },
+  mainnet: {
+    ...commonAssets,
+    RUNE: PoolAsset.RUNEB1A
+  },
+  chaosnet: {
```

Comment:
[suggestion] I think we can remove `chaosnet` to avoid duplication. And handle it properly in `mapNetworkToPoolAssets` then...

Reply:
imo its better to have explicit setting of values. in general the fact values are duplicated now its just a coincidence... am i wrong ?

Reply:
Assets (or `PoolAssets`) at `chaosnet` will be the same as for  `mainnet`.